### PR TITLE
Add country component

### PR DESCRIPTION
**Pull request by YiruDing**

### CODE REVIEW

File: client/components/Home.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Home = () => {
               <button>All Snacks</button>
             </Link>
             <Link to="/countries">
-              <button>All Countries</button>
+              <button>Add A Country</button>
```
Comment (alejvillanueva) [inline client/components/Home.js:18]: Hm this should not be on the home page. It should be a tab when you log in like the manage users and orders

File: client/components/Main.js [hunk 2]
```diff
@@ -28,6 +29,7 @@ class Main extends React.Component {
           <Switch>
             <Route exact path="/" component={Home}></Route>
             <Route exact path="/products" component={AllProducts} />
+            <Route exact path="/countries" component={CrearteCountry} />
```
Comment (alejvillanueva) [inline client/components/Main.js:32]: let's change it to manage-countries for consistency

File: client/components/ManageCountries/CrearteCountry.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        <h3>Add A Country:</h3>
+        <form onSubmit={onSave} action="/">
+          <label htmlFor="title">Name*:</label>
+          <input name="name" value={name} onChange={onChange} />
+          <br />
+          <label htmlFor="flag">Flag*:</label>
+          <input name="flag" value={flag} onChange={onChange} />
+          <br />
+          <label htmlFor="latitude">Latitude*:</label>
+          <input name="latitude" value={latitude} onChange={onChange} />
```
Comment (alejvillanueva) [inline client/components/ManageCountries/CrearteCountry.js:60]: add type="number" for the lat and long so they can only enter numbers

File: server/api/countries.js [hunk 4]
```diff
@@ -50,14 +50,14 @@ router.post('/', async (req, res, next) => {
...
-//         const updated = await countryToModify.update(req.body)
-//         res.status(200).send(updated)
-//     } catch (error) {
-//         next(error)
-//     }
-// })
+router.put('/:id', async (req, res, next) => {
+  try {
+    const countryToModify = await Country.findByPk(req.params.id);
+    const updated = await countryToModify.update(req.body);
```
Comment (alejvillanueva) [inline server/api/countries.js:56]: you should destructor this: 
 const {name, lat, lon} = req.body;
then  past them into line 56

File: client/components/Main.js (hunk 2 above)
Comment (YiruDing) [reply client/components/Main.js:32]: Done!

File: client/components/Home.js (hunk 1 above)
Comment (YiruDing) [reply client/components/Home.js:18]: Ok,I will leave it to Russel

File: client/components/ManageCountries/CrearteCountry.js (hunk 3 above)
Comment (YiruDing) [reply client/components/ManageCountries/CrearteCountry.js:60]: Done

File: server/api/countries.js (hunk 4 above)
Comment (YiruDing) [reply server/api/countries.js:56]: Done!Could you take a look plz??

**Review by alejvillanueva — APPROVED** [review]

**Pull request merged**